src/index.js

```diff
@@ -1,6 +1,16 @@
 const fibonacci = (n) => {
-
+  const fibonacciChallenge = [];
+  let a = 1;
+  let b = 1;
```

Comment:
use more descriptive variable names for function they will perform

Reply:
Thanks for your feeb back, I've changed variables name changed by the function they perform.